spring-cloud-contract-verifier/src/main/groovy/org/springframework/cloud/contract/verifier/builder/WebTestClientJUnitMethodBodyBuilder.groovy

```diff
@@ -0,0 +1,47 @@
+package org.springframework.cloud.contract.verifier.builder
+
+import groovy.transform.CompileStatic
+import groovy.transform.PackageScope
+import org.springframework.cloud.contract.spec.Contract
+import org.springframework.cloud.contract.spec.internal.Url
+import org.springframework.cloud.contract.verifier.config.ContractVerifierConfigProperties
+
+/**
+ * @author Olga Maciaszek-Sharma
+ *
+ * A {@link JUnitMethodBodyBuilder} implementation that uses WebTestClient to send requests.
+ *
+ * @author Olga Maciaszek-Sharma
+ *
+ * @since 2.1.0
+ */
+@CompileStatic
+@PackageScope
+class WebTestClientJUnitMethodBodyBuilder extends RestAssuredJUnitMethodBodyBuilder {
+
+    WebTestClientJUnitMethodBodyBuilder(Contract stubDefinition, ContractVerifierConfigProperties configProperties) {
+        super(stubDefinition, configProperties)
+    }
+
+    @Override
+    protected String returnedResponseType() {
+        return 'WebTestClientResponse'
```

Comment:
Why, oh why do you use single `'` ? :( :( :(

Reply:
Cause I like it.

Reply:
And cause we had a crazy mixture of both.